docs/application/native/guides/multimedia/thumbnail-images.md

```diff
@@ -81,76 +83,41 @@ To enable your application to use the thumbnail util functionality:
-
-   ```
-   ret = thumbnail_util_set_path(g_thumb_h, image_test_path);
-   ```
-
-   The function binds the thumbnail utility handle (first parameter) with the file specified in the path variable (second parameter).
-
-3. Set the thumbnail size using the `thumbnail_util_set_size()` function:
+1. To receive the results in a file, use the `thumbnail_util_extract_to_file()` function:
 
    ```
-   int width = 512;
-   int height = 288;
-
-   ret = thumbnail_util_set_size(g_thumb_h, width, height);
+   ret = thumbnail_util_extract_to_file(image_test_path, 512, 288, out_test_path);
    ```
 
-   If you do not set the size, the Thumbnail Util API use a default size of 320 x 240.
+   You find a thumbnail file on `out_test_path`.
 
-4. Extract the thumbnail from the file using the `thumbnail_util_extract()` function. As parameters, define the thumbnail utility handle, a callback for checking the extraction result, user data, and a request ID.
+2. To receive the results in a raw data, use the `thumbnail_util_extract_to_buffer()` function:
 
    ```
-   void
-   thumbnail_completed_cb(thumbnail_util_error_e error, const char *request_id,
-                          int raw_width, int raw_height, unsigned char *raw_data,
-                          int raw_size, void *user_data)
-   {
-       dlog_print(DLOG_DEBUG, LOG_TAG, "REQUEST ID: %s\n", request_id);
-       dlog_print(DLOG_DEBUG, LOG_TAG, "WIDTH: %d, HEIGHT: %d\n", raw_width, raw_height);
-   }
-
-   void
-   extract()
-   {
-       char *request_id = NULL;
-
-       ret = thumbnail_util_extract(g_thumb_h, thumbnail_completed_cb, NULL, &request_id);
-   }
+   unsigned char *out_buf;
+   size_t out_buf_size;
+   unsigned int out_width;
+   unsigned int out_height;
+   
+   ret = thumbnail_util_extract_to_buffer(image_test_path, 512, 288, &out_buf, &out_buf_size, &out_width, &out_height);
    ```
 
-   After calling the function, check whether the return value is `THUMBNAIL_UTIL_ERROR_NONE`. If it is, the function extracted the thumbnail successfully and stored it in the `raw_data` parameter returned by the callback. Otherwise, the function failed because of an error, which you need to handle.
-
-   The request ID is returned in the `request_id` parameter of the callback. You can use it to distinguish between different-sized thumbnails extracted from the same file, or to cancel a specific extraction request using the `thumbnail_util_cancel()` function.
-
-   The thumbnail is raw data that you can display on the screen directly. You can also encode and save the thumbnail to a file. When no longer needed, release the thumbnail using the `free()` function.
-
-5. When no longer needed, destroy the thumbnail utility handle using the `thumbnail_util_destroy()` function:
-
-   ```
-   thumbnail_util_destroy(g_thumb_h);
-   ```
+   You get a BGRA color image.
 
 ## Related Information
 - Dependencies
-  - Tizen 2.4 and Higher for Mobile
-  - Tizen 3.0 and Higher for Wearable
+  - Tizen 5.0 and Higher
```

Comment:
Hello, clockr63 

Would you distinguish the version information for Mobile and Wearable?
Like below

```
  - Tizen 5.0 and Higher for Mobile
  - Tizen 5.0 and Higher for Wearable
```

Also I think `Tizen 5.0 and Higher` can make an user confused 
as it sounds like Tizen 3.0 and 4.0 don't support thumbnail.

How about leaving it as `Tizen 3.0`, 
and add a note to updated part for Tizen 5.0, like "this functions is supported from 5.0"?

Thanks in advance.

Reply:
All existing functions have been deprecated. 
This guide has only new functions(for 5.0).

And these functions support all profiles. 
So, there is no need to distinguish the profile.

Reply:
For the version information, in that case, 5.0 would be OK.

For the profile information,
would you consider the consistency with other guide pages?
That format is to show the profile information for whole guide pages,
and for now this thumbnail-images.md page only is using different form.